pkg/controller/controller.go

```diff
@@ -79,12 +86,13 @@ type ErrorDetails struct {
 
 var (
 	categories = []string{"explanation", "breakglass"}
-	ttls       = []string{"15", "30", "60", "120", "240"}
+	ttls       = []string{"15m", "30m", "1h", "2h", "4h"}
```

Comment:
This is more general advice than a specific change request. Slices (and maps) are mutable in Go, so this kind of design lends itself to some difficult-to-debug edge cases: https://go.dev/play/p/A7JtU5oznsT. See "A possible gotcha" in https://go.dev/blog/slices-intro.

For this reason, especially for smaller structures like there, I recommend having a function that returns a copy:

```go
func ttls() []string {
  return []string{"15m", ...}
}
```

Note, do _not_ do this. Since slices are passed by reference, this is the same as just having the variable:

```go
# NO, BAD
var _ttls = []string{"15m", ...}

func ttls() []string {
  return ttls
}
```

Reply:
oh thanks for sharing, ill update for this